packages/checkout-ui-extensions-react/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^5.5.3",
-    "@quilted/react-testing": "^0.3.7",
+    "@quilted/react-testing": "^0.4.10",
```

Comment:
@jffortier It looks like this has introduced a breaking change in the tests for `ui-extensions`.

I tested reverting this back and the tests pass after that. Any ideas?

Reply:
This the error we get: 
```
 FAIL   checkout-ui-extensions-react  packages/checkout-ui-extensions-react/src/hooks/tests/metafields.test.tsx
  ● Test suite failed to run

    Cannot find module '@quilted/react-testing' from 'src/hooks/tests/mount.tsx'

    Require stack:
      src/hooks/tests/mount.tsx
      src/hooks/tests/metafields.test.tsx

      1 | import React from 'react';
    > 2 | import {createMount} from '@quilted/react-testing';
        | ^
      3 | import {
      4 |   ApiForRenderExtension,
      5 |   RenderExtensionPoint,

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (src/hooks/tests/mount.tsx:2:1)
```